frontend/src/lib/components/TaxonomicFilter/TaxonomicFilter.tsx

```diff
@@ -51,6 +51,7 @@ export function TaxonomicFilter({
     }, [])
 
     const placeholder = taxonomicGroupTypes
+        .filter(type => !type.includes('groups'))
```

Comment:
filtering this out for now since it's appending weird names like `groups_0`, `groups_1`